src/subscription/data/details/reducer.js

```diff
@@ -18,6 +18,7 @@ const subscriptionDetailsInitialState = {
   certificateType: 'verified',
   organization: 'University of California, Berkeley',
   price: 49,
+  paymentMethod: 'stripe',
```

Comment:
Do you know if we finalized the discussion, whether we'll get this from `subscription` BE or will we hard code it in the FE?